bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetThermoregulationHandler.java

```diff
@@ -0,0 +1,375 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.openwebnet.handler;
+
+import static org.openhab.binding.openwebnet.OpenWebNetBindingConstants.*;
+import static org.openhab.core.library.unit.SIUnits.CELSIUS;
+
+import java.math.BigDecimal;
+import java.util.Set;
+
+import javax.measure.Unit;
+import javax.measure.quantity.Temperature;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.openwebnet.OpenWebNetBindingConstants;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusInfo;
+import org.openhab.core.thing.ThingTypeUID;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.UnDefType;
+import org.openwebnet4j.communication.OWNException;
+import org.openwebnet4j.message.BaseOpenMessage;
+import org.openwebnet4j.message.FrameException;
+import org.openwebnet4j.message.MalformedFrameException;
+import org.openwebnet4j.message.Thermoregulation;
+import org.openwebnet4j.message.Thermoregulation.FAN_COIL_SPEED;
+import org.openwebnet4j.message.Thermoregulation.FUNCTION;
+import org.openwebnet4j.message.Thermoregulation.OPERATION_MODE;
+import org.openwebnet4j.message.Thermoregulation.VALVE_OR_ACTUATOR_STATUS;
+import org.openwebnet4j.message.Thermoregulation.WHAT;
+import org.openwebnet4j.message.Where;
+import org.openwebnet4j.message.WhereThermo;
+import org.openwebnet4j.message.Who;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link OpenWebNetThermoregulationHandler} is responsible for handling commands/messages for a Thermoregulation
+ * OpenWebNet device. It extends the abstract {@link OpenWebNetThingHandler}.
+ *
+ * @author Massimo Valla - Initial contribution
+ * @author Andrea Conte - Thermoregulation
+ * @author Gilberto Cocchi - Thermoregulation
+ */
+@NonNullByDefault
+public class OpenWebNetThermoregulationHandler extends OpenWebNetThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(OpenWebNetThermoregulationHandler.class);
+
+    public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES = OpenWebNetBindingConstants.THERMOREGULATION_SUPPORTED_THING_TYPES;
+
+    private boolean isTempSensor = false; // is the device a sensor or thermostat?
+
+    private Double currentSetPointTemp = 11.5d; // 11.5 is the default setTemp used in MyHomeUP mobile app
```

Comment:
Is this changed anywhere or can this be a constant?

Reply:
11.5°C is the default set-point temperature defined in the OpenWebNet documentation for thermostats.
It will change according to what you set up in your physical thermostat.